api/src/test/resources/testAppDataDir/configuration/jsonkeyvalues/config.json

```diff
@@ -21,5 +21,8 @@
     "report.opdconsult.active" : "true",
     "report.opdconsult.diagnosisQuestion.concept" : "95312123-e0c2-466d-b6b1-cb6e990d0d65",
     "report.opdconsult.diagnosesList.concept" : "b3ea209f-9979-481b-9c64-b6dd883c75bf",
-    "report.opdconsult.referredTo.concept" : "96408258-000b-424e-af1a-403919332938"
+    "report.opdconsult.referredTo.concept" : "96408258-000b-424e-af1a-403919332938",
+    "report.newEpisodesOfDiseases.conceptSet" : "c1ee4e1a-96d5-4cb1-82d1-95e367cfee20",
```

Comment:
Could you update the current [README](./README.md) and document the expected structure for this concept set.
1. Update the root [README](./README.md) to add this report to it.
2. Point to a local README (maybe located in the test folder of this report?) that specifically explains how the set up of this report's concepts should be done.

Reply:
Ok, thanks.

Reply:
@Ruhanga should we rename this to `report.newEpisodesOfDiseases.diseasesList.conceptSet`?

Reply:
This is more definitive, thanks @rbuisson. I'll add the change to this effect.